examples/complete/variables.tf

```diff
@@ -6,18 +6,6 @@ variable "availability_zone" {
   type = string
 }
 
-variable "namespace" {
-  type = string
-}
-
-variable "name" {
-  type = string
-}
-
-variable "stage" {
-  type = string
-}
-
```

Comment:
@syphernl 
since you added `context.tf`, please also remove `enabled` var from the module and from the example 

```
TestExamplesComplete 2020-10-19T17:02:14Z command.go:121: Error: Duplicate variable declaration
TestExamplesComplete 2020-10-19T17:02:14Z command.go:121: 
TestExamplesComplete 2020-10-19T17:02:14Z command.go:121:   on ../../variables.tf line 153:
TestExamplesComplete 2020-10-19T17:02:14Z command.go:121:  153: variable "enabled" {
TestExamplesComplete 2020-10-19T17:02:14Z command.go:121: 
TestExamplesComplete 2020-10-19T17:02:14Z command.go:121: A variable named "enabled" was already declared at ../../context.tf:80,1-19.
TestExamplesComplete 2020-10-19T17:02:14Z command.go:121: Variable names must be unique within a module.
```

`context.tf` has it so we don't have to re-define it in all places

thanks

Reply:
The `enabled` var has been removed from the module.